### DISCUSSION TITLE
How to connect on-premise network tap into AWS sensor node?

### DISCUSSION THREAD

**Discussion by ManagedGuard** [Security-Onion-Solutions/securityonion#8761] [Unsupported Versions]
I wanted to use AWS Servers to manage security onion, but i dont know how to configure or connect on-premise networks (Networks & Wazuh Agent Endpoints) into AWS sensor, help me please?

**Comment by dougburks** [comment]
Have you checked our documentation? It includes some links which might help:
https://docs.securityonion.net/en/2.3/cloud-ami.html
